src/EnergyPlus/HybridUnitaryAirConditioners.cc

```diff
@@ -0,0 +1,728 @@
+				SetupOutputVariable("Zone Hybrid Unitary HVAC Humidification Load to Humidistat Setpoint Heat Tansfer Energy", OutputProcessor::Unit::J, ZoneHybridUnitaryAirConditioner(UnitLoop).RequestedHumdificationEnergy, "System", "Average", ZoneHybridUnitaryAirConditioner(UnitLoop).Name);
+
+
+				SetupOutputVariable("Zone Hybrid Unitary HVAC Supply Fan Electric Power", OutputProcessor::Unit::W, ZoneHybridUnitaryAirConditioner(UnitLoop).SupplyFanElectricPower, "System", "Average", ZoneHybridUnitaryAirConditioner(UnitLoop).Name);
+				SetupOutputVariable("Zone Hybrid Unitary HVAC Supply Fan Electric Energy", OutputProcessor::Unit::J, ZoneHybridUnitaryAirConditioner(UnitLoop).SupplyFanElectricEnergy, "System", "Sum", ZoneHybridUnitaryAirConditioner(UnitLoop).Name, _, "Electric", "Fans", "Hybrid HVAC Fans", "System");
+				SetupOutputVariable("Zone Hybrid Unitary HVAC Secondary Fuel Consumption Rate", OutputProcessor::Unit::W, ZoneHybridUnitaryAirConditioner(UnitLoop).SecondaryFuelConsumptionRate, "System", "Average", ZoneHybridUnitaryAirConditioner(UnitLoop).Name);
+				SetupOutputVariable("Zone Hybrid Unitary HVAC Secondary Fuel Consumption", OutputProcessor::Unit::J, ZoneHybridUnitaryAirConditioner(UnitLoop).SecondaryFuelConsumption, "System", "Average", ZoneHybridUnitaryAirConditioner(UnitLoop).Name);
+				SetupOutputVariable("Zone Hybrid Unitary HVAC Third Fuel Consumption Rate", OutputProcessor::Unit::W, ZoneHybridUnitaryAirConditioner(UnitLoop).ThirdFuelConsumptionRate, "System", "Average", ZoneHybridUnitaryAirConditioner(UnitLoop).Name);
+				SetupOutputVariable("Zone Hybrid Unitary HVAC Third Fuel Consumption", OutputProcessor::Unit::J, ZoneHybridUnitaryAirConditioner(UnitLoop).ThirdFuelConsumption, "System", "Average", ZoneHybridUnitaryAirConditioner(UnitLoop).Name);
+				 SetupOutputVariable("Zone Hybrid Unitary HVAC Water Consumption Rate", OutputProcessor::Unit::kgWater_s, ZoneHybridUnitaryAirConditioner(UnitLoop).WaterConsumptionRate, "System", "Average", ZoneHybridUnitaryAirConditioner(UnitLoop).Name);
+				//make sure units above are right.
+				 SetupOutputVariable("Zone Hybrid Unitary HVAC Water Consumption", OutputProcessor::Unit::m3, ZoneHybridUnitaryAirConditioner(UnitLoop).WaterConsumption, "System", "Average", ZoneHybridUnitaryAirConditioner(UnitLoop).Name);
+				SetupOutputVariable("Zone Hybrid Unitary HVAC External Static Pressure", OutputProcessor::Unit::Pa, ZoneHybridUnitaryAirConditioner(UnitLoop).ExternalStaticPressure, "System", "Average", ZoneHybridUnitaryAirConditioner(UnitLoop).Name);
+
+				std::vector<CSetting>::iterator iterOperatingSettings;
+				int index=0;
+				int size= ZoneHybridUnitaryAirConditioner(UnitLoop).CurrentOperatingSettings.size();
+
+				for (int i = 0; i< size; i++)
+				{
+					CSetting *po = &ZoneHybridUnitaryAirConditioner(UnitLoop).CurrentOperatingSettings[i];			

```

Comment:
This isn't necessary, right?  If you want to create a reference, keep it simple:

```
auto & thisSetting = ZoneHybridUnitaryAirConditioner(UnitLoop).CurrentOperatingSettings[i];
```

And in the string variable names, just use plain C++ concatenation instead of creating a string stream object.

That will make this section a lot cleaner.

Reply:
ok done